src/transformers/models/informer/modeling_informer.py

```diff
@@ -647,7 +647,7 @@ def forward(
         # calculate context for updating the attn_output, based on:
         # https://github.com/zhouhaoyi/Informer2020/blob/ac59c7447135473fb2aafeafe94395f884d5c7a5/models/attn.py#L74
         if self.is_decoder:
-            context = value_states.cumsum(dim=-2)
+            context = value_states.cumsum(dim=-2, dtype=torch.float32).to(value_states.dtype)
```

Comment:
```suggestion
            context = value_states.cumsum(dim=-2, dtype=value_states.dtype)
```

Reply:
so I wanted the cumsum to be in float32 to avoid overflow etc. and then cast...